lib/discourse_slack/slack.rb

```diff
@@ -120,10 +120,20 @@ def self.get_key(id = nil)
     def self.set_filter_by_id(id, channel, filter, tags = nil, channel_id = nil)
       data = get_store(id)
```

Comment:
I haven't investigated the complexity of making this type of change, but in the future it'd probably make sense in the future for the ID to be the channel name.